test/e2e/specs/site-editor/site-editor-inserter.spec.js

```diff
@@ -64,4 +65,199 @@ test.describe( 'Site Editor Inserter', () => {
 
 		await expect( blockLibrary ).toBeHidden();
 	} );
+
+	test( 'should open the inserter to patterns tab if using zoom out', async ( {
+		InserterUtils,
+	} ) => {
+		const zoomOutButton = InserterUtils.getZoomOutButton();
+		const inserterButton = InserterUtils.getInserterButton();
+		const blockLibrary = InserterUtils.getBlockLibrary();
+
+		await zoomOutButton.click();
+		await expect( await InserterUtils.getZoomCanvas() ).toBeVisible();
+
+		await inserterButton.click();
+		const patternsTab = InserterUtils.getBlockLibraryTab( 'Patterns' );
+		await expect( patternsTab ).toHaveAttribute(
+			'data-active-item',
+			'true'
+		);
+		await expect( await InserterUtils.getZoomCanvas() ).toBeVisible();
```

Comment:
Nit: this feels a bit cumbersome. Moreover, this is only ever used for asserting on the active state of Zoom Out rather than retrieving a reference to the Zoom Out canvas itself. 

Therefore it might be a clearer to have an API something like this (maybe via a dedicated ZoomOut helper?):

```suggestion
		await expect( await ZoomOutUtils.isActive() );
```

Not a deal breaker but I think it would improve the readability of the tests.